p2p-specs/p2p-interface.md

```diff
@@ -405,7 +412,8 @@ The response MUST consist of a single `response_chunk`.
 
 Clients SHOULD immediately disconnect from one another following the handshake above under the following conditions:
 
-1. If the `supported_mempools` and client's own list of supported mempools are disjoint.
+1. If `chain_id` does not match the node's local `chain_id`, since the peer is on another network.
+2. Clients MAY choose to disconnect from peers whose `block_number` is sufficiently behind their local block number. Implementers are free to implement such behavior in their own way. Note that some amount of leniency is suggested as nodes discover new blocks at different rates.
```

Comment:
Should we suggest a number here? Mostly as a suggestion. Otherwise we might end up with arbitrary disconnects.